index.js

```diff
@@ -31,10 +35,11 @@ module.exports = function packager (opts, cb) {
     default: return cb(new Error('Unsupported platform. Must be either darwin, linux, or win32'))
   }
 
-  // Ignore this and related modules by default
-  var defaultIgnores = ['/node_modules/electron-prebuilt($|/)', '/node_modules/electron-packager($|/)', '/\.git($|/)']
-  if (opts.ignore && !Array.isArray(opts.ignore)) opts.ignore = [opts.ignore]
-  opts.ignore = (opts.ignore) ? opts.ignore.concat(defaultIgnores) : defaultIgnores
+  // Only include files specified
+  var defaultFiles = ['package.json']
+  opts.files = (defaultFiles + config.files).map(function (item) {
+    path.join(opts.dir, item)
```

Comment:
Yeah, I'm finding a lot of errors just by running things in my env. I'll
update this PR as the day goes.

On Sat, Jun 20, 2015 at 12:11 PM, maxogden <notifications@github.com> wrote:

> In index.js
> <https://github.com/maxogden/electron-packager/pull/80#discussion_r32886107>
> :
>
> > @@ -31,10 +35,11 @@ module.exports = function packager (opts, cb) {
> >      default: return cb(new Error('Unsupported platform. Must be either darwin, linux, or win32'))
> >    }
> >
> > -  // Ignore this and related modules by default
> > -  var defaultIgnores = ['/node_modules/electron-prebuilt($|/)', '/node_modules/electron-packager($|/)', '/\.git($|/)']
> > -  if (opts.ignore && !Array.isArray(opts.ignore)) opts.ignore = [opts.ignore]
> > -  opts.ignore = (opts.ignore) ? opts.ignore.concat(defaultIgnores) : defaultIgnores
> > +  // Only include files specified
> > +  var defaultFiles = ['package.json']
> > +  opts.files = (defaultFiles + config.files).map(function (item) {
> > +    path.join(opts.dir, item)
>
> I think you need a return here
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/maxogden/electron-packager/pull/80/files#r32886107>.
>



-- 
Kurtis Rainbolt-Greene, Hacker
Software Developer
1631 8th St.
New Orleans, LA
70115